src/github.js

```diff
@@ -5,6 +5,22 @@ export default class GitHub {
 	constructor() {
 		const token = core.getInput("token") || process.env.GITHUB_TOKEN || "";
 		this.octokit = github.getOctokit(token);
+
+		const formats = core.getInput("format").replace(/ /g, '').split(',').filter( value => {
+			return value.trim();
+		});
+
+
+		if ( formats.length === 0 ) {
+			this.format = [ "git" ];
+		}
+		else if ( formats.includes('all') ) {
```

Comment:
This `else if` is on a new line, but the ones below are not. We should make them all consistent.